.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
Enables the "Don’t see your issue here? Open a blank issue." link at the bottom of the page

Reply:
That did not work last time I tested, hopefully, it does now

Reply:
I've tried these in a private test repo and they showed up as you can see in the screenshot above.